Client/metrics.yaml

```
@@ -850,3 +850,40 @@ pocket:
     notification_emails:
       - firefox-ios@mozilla.com
     expires: "2021-08-19"
+
+# App menu
+app_menu:
+  library:
+    type: counter
+    description: |
+        Counts the number of times a user taps Your Library in the app menu
+    bugs:
+      - https://github.com/mozilla-mobile/firefox-ios/issues/1544
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-ios/pull/8167
+    notification_emails:
+      - firefox-ios@mozilla.com
+    expires: "2021-08-17"
```

Shouldn't this be 6 months from today i.e. Sepetember 17th as you want this for 6 months 
```suggestion
    expires: "2021-09-17"
```

---

Ya I added wrong :)